tests/compile-fail/mem_forget.rs

```diff
@@ -0,0 +1,21 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+use std::sync::Arc;
+
+use std::mem::forget as forgetSomething;
+use std::mem as memstuff;
+
+#[deny(mem_forget)]
+fn main() {
+    let five: i32 = 5;
+    forgetSomething(five);
+    //~^ ERROR usage of std::mem::forget
+
+    let six: Arc<i32> = Arc::new(6);
+    memstuff::forget(six);
+    //~^ ERROR usage of std::mem::forget
+
+    std::mem::forget(7);
```

Comment:
I limited it to `Drop` types, but it seems like `get_trait_def_id(cx, &paths::DROP)` is returning `None`, preventing it from working. Any idea why?